app/src/main/java/com/example/nicocalculator/mvp/presenter/Presenter.kt

```diff
@@ -0,0 +1,117 @@
+package com.example.nicocalculator.mvp.presenter
+
+import android.widget.Button
+import com.example.nicocalculator.Key
+import com.example.nicocalculator.MainActivity
+import com.example.nicocalculator.mvp.model.CalculatorModel
+import com.example.nicocalculator.mvp.view.KeyView
+
+class Presenter(private val model: CalculatorModel, private val view: KeyView) {
```

Comment:
Please try to use the same name for the presenter, model and view. In this case, I think that can be CalculatorPresenter, CalculatorModel and CalculatorView

Reply:
Done